tendrl/integrations/ceph/objects/journal/__init__.py

```diff
@@ -0,0 +1,20 @@
+from tendrl.commons import objects
+
+
+class Journal(objects.BaseObject):
+    internal = True
+    def __init__(self, integration_id, node_id, data, *args, **kwargs):
+        self._defs = {}
+        super(Journal, self).__init__(*args, **kwargs)
+
+        self.integration_id = integration_id
+        self.node_id = node_id
+        self.data = data
```

Comment:
Per OSD journal details are journal metadata. This information I am maintaining per host for next time journal mapping process, so not more journals get created on SSD disks.

This is nothing to do with journal details of OSDs in ceph cluster.